packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -21,13 +33,46 @@ class Chat extends Widget<Properties> {
     $(this.element()).addClass(CHAT_CLASS);
 
     super._initMarkup();
+
+    this._renderHeader();
+    this._renderMessageList();
+    this._renderMessageBox();
+  }
+
+  _renderHeader(): void {
+    const { title } = this.option();
+
+    this._chatHeader = this._createComponent($('<div>'), ChatHeader, { title });
```

Comment:
The same https://github.com/DevExpress/DevExtreme/pull/27815/files#r1689329383

Reply:
[5d815c8cb9c08ceb8edcac2cefb7e4c48907db00]